docs/source/contact.rst

```diff
@@ -0,0 +1,22 @@
+Contact
+===================================
+
+Are you missing something? Would you like to contribute in this documentation?
+Contact any of us:
+
+.. list-table:: Title
+   :widths: 25 75
+   :header-rows: 1
+
+   * - Contact Person
+     - Organisation Task
+   * - `Jose Angel Hernando <jose.hernando@usc.es>`_
```

Comment:
I can't see the names, can you?

Reply:
No, it's fine, I was just wondering why I couldn't visualize the emails although they are written in the `.rst`. I can approve the PR as it is, anyway.